conda-forge.yml

```diff
@@ -1,5 +1,6 @@
 channel_priority: strict
```

Comment:
Don't modify the top-level conda-forge.yml, place one on the recipe dir.

Reply:
Good call. Thanks.